subsys/bluetooth/controller/hci_internal.c

```diff
@@ -979,6 +995,29 @@ static uint8_t le_controller_cmd_put(uint8_t const * const cmd,
 		return sdc_hci_cmd_le_read_antenna_information((void *)event_out_params);
 #endif
 
+#if defined(CONFIG_BT_CTLR_LE_POWER_CONTROL)
+	case SDC_HCI_OPCODE_CMD_LE_ENHANCED_READ_TRANSMIT_POWER_LEVEL:
+		*param_length_out +=
+			sizeof(sdc_hci_cmd_le_enhanced_read_transmit_power_level_return_t);
+		return sdc_hci_cmd_le_enhanced_read_transmit_power_level((void *)cmd_params,
+									 (void *)event_out_params);
+
+	case SDC_HCI_OPCODE_CMD_LE_READ_REMOTE_TRANSMIT_POWER_LEVEL:
+		return sdc_hci_cmd_le_read_remote_transmit_power_level((void *)cmd_params);
+
+	case SDC_HCI_OPCODE_CMD_LE_SET_TRANSMIT_POWER_REPORTING_ENABLE:
+		*param_length_out += sizeof(sdc_hci_cmd_le_set_transmit_power_reporting_enable_return_t);
+		return sdc_hci_cmd_le_set_transmit_power_reporting_enable((void *)cmd_params,
+									  (void *)event_out_params);
+
+	case SDC_HCI_OPCODE_CMD_LE_READ_RF_PATH_COMPENSATION:
```

Comment:
`RF_PATH_COMPENSATION `commands are not fully implemented. They are excluded in [supported commands ](https://github.com/nrfconnect/sdk-nrf/pull/9182/files#diff-f5bf9d2614b19c06ca472417c8004daeae8125af4a59c8ec6983180c9ddae3daR382) . But they are still useful in certain LL conformance tests for LE Power Control feature.

Reply:
Did you mean "included" instead of "excluded"? They seem to be enabled on the line you linked to.

Reply:
No i did not enable. I did not wanted to add as `RF_PATH_COMPENSATION ` supported when it is read with `Read Local Supported Commands` HCI command. So following lines are missing

> cmds->hci_le_read_rf_path_compensation = 1;
> cmds->hci_le_write_rf_path_compensation = 1;

Reply:
Ok, then it was just me misreading the feature names.